swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -194,19 +185,25 @@ Now, as all the infrastructure required are up and running and configured, start
 import ballerina/log;
 import ballerinax/azure_functions as af;
 
-@af:Function
-public function cosmosDBToQueue2(@af:CosmosDBTrigger { 
-        connectionStringSetting: "CosmosDBConnection", databaseName: "db1", 
-        collectionName: "c1" } json req,
-        @af:QueueOutput { queueName: "queue2" } af:StringOutputBinding outMsg) returns error?{
-    json[] entryList = <json[]> req;
-    string name = check entryList[0].name;
-    log:printInfo(name);
-    outMsg.value = name;
+public type DBEntry record {
+    string id;
+    string name;
+};
+
+@af:CosmosDBTrigger {connectionStringSetting: "CosmosDBConnection", databaseName: "db1", collectionName: "c1"}
+listener af:CosmosDBListener cosmosEp = new ();
+
+service "cosmos" on cosmosEp {
+    remote function onUpdated(DBEntry[] entries) returns @af:QueueOutput {queueName: "people"} string {
+        string name = entries[0].name;
+        log:printInfo(entries.toJsonString());
+        return "Hello, " + name;
+    }
 }
+
 ```
 
-Build the package by executing the `bal build` command on the package directory, and then deploy it using the `az cli` command shown in the Ballerina build output as in the previous section.
+Build the package by executing the `bal build` command on the package directory, and then deploy it using the `func` command shown in the Ballerina build output as in the previous section.
```

Comment:
```suggestion
Execute the `bal build` command on the package directory to build the package. Then, execute the `func` command shown in the Ballerina build output as in the previous section to deploy it.
```

Better if we can mention the full command here rather than asking to refer the previous section.